File: RedSheet.DbEntities/Models/vArrivalAndOpeningTactic.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vArrivalAndOpeningTactics", Schema = "dbo")]
	[Serializable]
	public partial class vArrivalAndOpeningTactic  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ArrivalAndOpeningTacticId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string ArrivalAndOpeningTacticValue { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectEventTimelineId { get; set; } 

	public vArrivalAndOpeningTactic()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-arrival-and-opening-tactic.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vArrivalAndOpeningTactic {
    constructor(vArrivalAndOpeningTactic?: vArrivalAndOpeningTactic )  {
        let properties = [ "arrivalAndOpeningTacticId", "arrivalAndOpeningTacticValue", "projectEventTimelineId",];
        for (let property of properties)
            if (vArrivalAndOpeningTactic && vArrivalAndOpeningTactic[property])
                this[property] = vArrivalAndOpeningTactic[property];
    }
 
	arrivalAndOpeningTacticId : number =   0 ;
 
    @required()
    @maxLength(1000)
	arrivalAndOpeningTacticValue : string =   undefined;
 
    @range(1,2147483647)
	projectEventTimelineId : number =   undefined;


}
